synapse/storage/databases/main/roommember.py

```diff
@@ -861,97 +860,54 @@ async def get_mutual_rooms_between_users(
 
         return shared_room_ids or frozenset()
 
-    async def get_joined_user_ids_from_state(
-        self, room_id: str, state: StateMap[str], state_entry: "_StateCacheEntry"
-    ) -> Set[str]:
-        state_group: Union[object, int] = state_entry.state_group
-        if not state_group:
-            # If state_group is None it means it has yet to be assigned a
-            # state group, i.e. we need to make sure that calls with a state_group
-            # of None don't hit previous cached calls with a None state_group.
-            # To do this we set the state_group to a new object as object() != object()
-            state_group = object()
+    async def get_joined_users_from_state(
+        self, room_id: str, state: StateMap[str]
+    ) -> Dict[str, ProfileInfo]:
```

Comment:
It looks like the merge has gone awry here: we're now reverting `get_joined_user_ids_from_state` back to `get_joined_users_from_state` (cf https://github.com/matrix-org/synapse/pull/13573)

Reply:
Another 🤦 for me today - https://github.com/matrix-org/synapse/pull/13569/commits/64044817b4d968a74fdd6e42f4372336c921eb4a (+ https://github.com/matrix-org/synapse/pull/13569/commits/4e62c4632baabda62e21ffff32bfb088407a80f4).